docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1231,22 +1246,22 @@ quarkus tls lets-encrypt renew-certificate \
 During this command, TLS registry CLI reads a Let's Encrypt account information recorded during the <<lets-encrypt-issue-certificate>> step, issues a Let's Encrypt certificate request, and communicates with a Quarkus application to have ACME challenges resolved.
 
 Once the Let's Encrypt certificate chain and private key have been successfully renewed, they are converted to PEM format and copied to your application's `.letsencrypt` folder.
-TLS registry is informed that a new certificate and private key are ready and it reloads them automatically.
+The TLS registry is informed that a new certificate and private key are ready and that they are reloaded automatically.
```

Comment:
Question: Think the updated text slightly alters the meaning. The original sentence states that the TLS registry **reloads** the new cert and private automatically, whereas your update states that the TLS registry **is informed** that they are reloaded (and not who does the reloading).

Reply:
@jmartisk WDYT, Jan?

Reply:
It should be that the TLS registry performs the 'reload'.

Reply:
Great.

Applying:
```
The TLS registry is notified when a new certificate and private key are ready, and it automatically reloads them.
```